test/linters/i18n_test.js

```diff
@@ -198,6 +198,88 @@ describe("I18nLinter", function() {
       });
     });
 
+    it("reports a success for pluralization keys not present in the default locale but present in others", function() {
+      mockFs({
+        "/path/to/theme": {
+          locales: {
+            "en.default.json": JSON.stringify({
+              products: {
+                product: {
+                  count: {
+                    one: "un",
+                    other: "autre"
+                  }
+                }
+              }
+            }),
+            "fr.json": JSON.stringify({
+              products: {
+                product: {
+                  count: {
+                    one: "un",
+                    many: "plusieurs",
+                    other: "autre"
+                  }
+                }
+              }
```

Comment:
I think it would make sense to change this from `fr`to a language that actually uses `many` (ie. `cs`).

Reply:
Same as above. I'll make note of that and update it in the future, since we already made the release